src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -391,9 +415,48 @@ void OverlayWindow::init_settings()
         {
             theme.value = std::move(*val);
         }
+        if (auto val = settings.get_string_value(disabledApps.name))
+        {
+            disabledApps.value = std::move(*val);
+            update_disabled_apps();
+        }
     }
     catch (std::exception&)
     {
         // Error while loading from the settings file. Just let default values stay as they are.
     }
 }
+
+bool OverlayWindow::is_disabled_app(wchar_t* exePath)
+{
+    auto exePathUpper = std::wstring(exePath);
+    CharUpperBuffW(exePathUpper.data(), (DWORD)exePathUpper.length());
+
+    for (const auto& row : disabled_apps_array)
+    {
+        const auto pos = exePathUpper.rfind(row);
+        const auto last_slash = exePathUpper.rfind('\\');
+        // Check that row occurs in disabled_apps_array, and its last occurrence contains in itself the first character after the last backslash.
+        if (pos != std::wstring::npos && pos <= last_slash + 1 && pos + row.length() > last_slash)
+        {
+            return true;
+        }
+    }
+    return false;
+}
+
+void OverlayWindow::update_disabled_apps()

```

Comment:
Same as above, this function can crash, it should have a try catch.

Reply:
I think this should be fine since by default `disabledApps.value` is an empty string and `disabled_apps_array` an empty vector.
Let me know if a try-catch is still needed.